third_party/docfx-doclet-143274/src/main/java/com/microsoft/build/PackageBuilder.java

```diff
@@ -35,16 +31,24 @@ class PackageBuilder {
     this.referenceBuilder = referenceBuilder;
   }
 
-  MetadataFile buildPackageMetadataFile(PackageElement packageElement) {
-    String fileName = packageLookup.extractHref(packageElement);
-    MetadataFile packageMetadataFile = new MetadataFile(outputPath, fileName);
-    MetadataFileItem packageItem =
-        new MetadataFileItem(LANGS, packageLookup.extractUid(packageElement));
-    packageItem.setId(packageLookup.extractId(packageElement));
-    referenceBuilder.addChildrenReferences(
-        packageElement, packageItem.getChildren(), packageMetadataFile.getReferences());
-    populateItemFields(packageItem, packageLookup, packageElement);
-    packageMetadataFile.getItems().add(packageItem);
-    return packageMetadataFile;
+  PackageOverviewFile buildPackageOverviewFile(
+      PackageElement packageElement,
+      RepoMetadata repoMetadata,
+      String artifactVersion,
+      String recommendedApiVersion) {
+    String status = packageLookup.extractStatus(packageElement);
+    String fileName = packageLookup.extractHref(packageElement).replace(".yml", ".md");
```

Comment:
Yep - the new Package Overview page is a markdown file (otherwise we'd have to change the way doc-pipeline renders .yml files to get the new overview, which was significantly more complex). I can see how this is unnecessarily confusing though so I've updated it to use the `packageElement` directly to generate the fileName.